tests/python/contrib/test_arm_compute_lib/test_conv2d.py

```diff
@@ -127,51 +241,31 @@ def test_conv2d():
 
     device = Device()
     np.random.seed(0)
+    r = random.Random(0)
```

Comment:
I'm not sure if it's a good idea to random shapes in unit tests. It would make the CI nondeterministic on different machine even the random seed is fixed.

cc @zhiics @tqchen

Reply:
The discussion in #6084 seems to suggest not relying on a seed

Reply:
Exactly, so I meant we should explicitly list some shapes here and only use them for testing instead of randomly generating a set of shapes.

Reply:
I'm wondering if this point could be extended to input as well. Currently I seed a random input, should this not be the case?

Reply:
Although it is not the best practice neither to randomly generate input tensors as you pointed out, it is relatively acceptable in general. The reasons are:

1. The module we are testing does not highly depend on the input values. 
2. It is tedious to put a constant large tensor (e.g., 1, 3, 224, 224 = 150k numbers) in the code base.
3. Sometimes people use `np.ones` or `np.zeros` to be inputs for the above two reasons, but these tensors are too far away from real world applications and may not fit our testing purposes.

Reply:
BTW, why do we need a random input? It looks we can just give the output channel a value or multiple values according to the number of trails, right?

Reply:
Makes sense, thanks for the explanation @comaniac. I'll change this to fixed shapes and channels. Yep that's correct @zhiics